### ISSUE TITLE
Undefined index: viewerHasStarred

### ISSUE THREAD

**Issue by PhilETaylor** [symfony/thanks#43]
When I run 

`composer thanks`

I get

![screen shot 2018-02-01 at 18 39 08](https://user-images.githubusercontent.com/400092/35696372-363d0008-077f-11e8-8c66-98bf6e21c31e.png)

What debugging info would you like?

`Composer version 1.6.2 2018-01-05 15:28:41`

**Comment by PhilETaylor** [comment]
```
 () at /Users/phil/.composer/vendor/symfony/thanks/src/Command/ThanksCommand.php:176
 Composer\Util\ErrorHandler::handle() at /Users/phil/.composer/vendor/symfony/thanks/src/Command/ThanksCommand.php:176
 Symfony\Thanks\Command\ThanksCommand->execute() at phar:///usr/local/bin/composer.phar/vendor/symfony/console/Command/Command.php:242
 Symfony\Component\Console\Command\Command->run() at phar:///usr/local/bin/composer.phar/vendor/symfony/console/Application.php:842
 Symfony\Component\Console\Application->doRunCommand() at phar:///usr/local/bin/composer.phar/vendor/symfony/console/Application.php:193
 Symfony\Component\Console\Application->doRun() at phar:///usr/local/bin/composer.phar/src/Composer/Console/Application.php:251
 Composer\Console\Application->doRun() at phar:///usr/local/bin/composer.phar/vendor/symfony/console/Application.php:117
 Symfony\Component\Console\Application->run() at phar:///usr/local/bin/composer.phar/src/Composer/Console/Application.php:100
 Composer\Console\Application->run() at phar:///usr/local/bin/composer.phar/bin/composer:58
 require() at /usr/local/bin/composer.phar:24
```

**Comment by PhilETaylor** [comment]
Looks like $repo at 
vendor/symfony/thanks/src/Command/ThanksCommand.php:176

is not actually a repo as when var_dump'ed it looks like this (truncated for gh) 

![screen shot 2018-02-01 at 18 45 07](https://user-images.githubusercontent.com/400092/35696691-0cbd1884-0780-11e8-8ded-a6fc81c02e36.png)

**Comment by nicolas-grekas** [comment]
Thanks for the detailed report.
I managed to reproduce the issue when using hirak/prestissimo. Is that your case?
If yes, then this should be fixed by #44

**Comment by PhilETaylor** [comment]
ah yes - I have hirak/prestissimo installed on all my machines. I'll test #44 when in the office tomorrow (in bed now)
